services/download/src/main/java/app/coronawarn/server/services/download/runner/Download.java

```diff
@@ -68,51 +71,54 @@
 
   @Override
   public void run(ApplicationArguments args) {
-    try {
-      // Download federation batches from day before
-      // process them, add batchTag from this response to federationbatch table
-
-      FederationBatch batchToProcess = federationBatchService.getNextFederationBatchToProcess();
-
-      if (batchToProcess == null) {
-        logger.debug("No (further) batches to process found in table.");
-        return;
-      }
-
-      List<FederationBatch> federationBatches =
-          federationBatchService.getFederationBatches();
-
-      for (FederationBatch federationBatch : federationBatches) {
-        try {
-          processFederationBatch(federationBatch);
-          federationBatchService.markFederationBatchAsProcessed(batchToProcess);
-        } catch (Exception e) {
-          // TODO: error handling for failure during handling of single federationBatch?
-          logger.error(e.getMessage());
-        }
-      }
-
-    } catch (Exception e) {
-      logger.error("Download of diagnosis key batch failed.", e);
-      Application.killApplication(applicationContext);
+    // run();
+  }
+
+  private void run() {
```

Comment:
Why the private `run`? Also, the call to this method is commented on the overridden `run`. Is that on purpose?